package.json

```diff
@@ -1,13 +1,16 @@
 {
   "license": "MIT",
   "scripts": {
-    "start": "node app.js",
+    "start": "npm-run-all --parallel watch:app watch:scss",
```

Comment:
so is start used when we deploy to production and also on CI?

`start:dev` rather than just `dev`? or you don't care.  I mention the point because most of our other apps have just `dev` and I was wondering if it would be good to keep it consistent